### PR TITLE
Alt Text Inclusions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing alt text.

### CODE REVIEW

File: iis/extensions/url-rewrite-module/setting-http-request-headers-and-iis-server-variables.md [hunk 1]
```diff
@@ -77,15 +77,15 @@ The next step is to define a rewrite map that will be used to map the URL part,
...
 
 Click "Add Rewrite Map..." and specify the name of the map as "Languages":
 
-![](setting-http-request-headers-and-iis-server-variables/_static/image20.png)
+![Screenshot of the Add Rewrite Map dialog box with Languages entered as the Rewrite map name.](setting-http-request-headers-and-iis-server-variables/_static/image20.png)
 
 This map will define mappings between the URL part that represents a language and the locale identifier to be used when setting the HTTP request cookie. Click "Edit Map Settings..." action to specify the default value to use when no mapping can be found. Enter "en\_US" as a default value:
 
-![](setting-http-request-headers-and-iis-server-variables/_static/image24.png)
+![Screenshot of the Edit Rewrite Map dialog box showing E N underscore U S as the default value to use when the key is not found in the map.](setting-http-request-headers-and-iis-server-variables/_static/image24.png)
```
Comment (MeeraDi) [inline iis/extensions/url-rewrite-module/setting-http-request-headers-and-iis-server-variables.md:88]: Suggestion: Screenshot of the Edit Rewrite Map dialog box showing E N underscore U S as the default value when the key is not found.

File: iis/extensions/url-rewrite-module/setting-http-request-headers-and-iis-server-variables.md [hunk 2]
```diff
@@ -136,6 +136,6 @@ To test that the rule correctly sets the cookie and a server variable, open a We
 
 `http://localhost/fr-fr/language.aspx`
 
-![](setting-http-request-headers-and-iis-server-variables/_static/image33.png)
+![Screenshot of the U R L Rewrite Module v 2 Test webpage with a focus on the U R L bar, the Language cookie being set to F R underscore F R, and the originally requested U R L.](setting-http-request-headers-and-iis-server-variables/_static/image33.png)
```
Comment (MeeraDi) [inline iis/extensions/url-rewrite-module/setting-http-request-headers-and-iis-server-variables.md:139]: Over 140 characters. Suggestion: Screenshot of the U R L Rewrite Module v 2 Test webpage with focus on the U R L bar, the Language cookie is set to F R underscore F R.

File: iis/publish/using-web-deploy/testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix.md [hunk 3]
```diff
@@ -20,33 +20,33 @@ So far, we have configured Web Deploy on a server so non-administrators can depl
...
 2. Click the ContsoUniversity Web Application Project (WAP)
 3. Click the Publish dropdown in the toolbar and click "New…"  
-    [![](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image10.png)](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image9.png)
+    [![Screenshot of the Publish drop-down menu with its New option being highlighted.](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image10.png)](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image9.png)
 4. Fill out the dialog like below:  
-    [![](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image12.png)](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image11.png)
+    [![Screenshot of the Publish Web dialog with the Publish method field set to the Web Deploy entry, and the site or application set to the Default Web Site entry.](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image12.png)](testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix/_static/image11.png)
```
Comment (MeeraDi) [inline iis/publish/using-web-deploy/testing-web-deploy-publishing-from-visual-studio-2010-and-webmatrix.md:45]: Over 140 characters. Suggestion: Screenshot of the Publish Web dialog. The Publish method field = Web Deploy entry, site or application = Default Web Site entry.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
See [the build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C23%5C90f071cf-c002-b629-5e11-5bc968bb67ca%5CPullRequest%5C202203231724579047-729%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-23T17%3A26%3A42Z&sr=b&sp=r&sig=2miUDMpdL98E11hDvvB3Lz5puTi3Gr13KrByyW4n1dM%3D) lots of missing alt text in iis/publish/using-the-ftp-service/using-fsrm-folder-quotas-with-ftp-in-iis-7.md

**Comment by dereklh7** [comment]
@Rick-Anderson The missing alt text in iis/publish/using-the-ftp-service/using-fsrm-folder-quotas-with-ftp-in-iis-7.md will be addressed in a separate PR.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**